docs/providers/aws/guide/functions.md

```diff
@@ -284,6 +284,27 @@ functions:
 
 During the first deployment when locally built images are used, Framework will automatically create a dedicated ECR repository to store these images, with name `serverless-<service>-<stage>`. Currently, the Framework will not remove older versions of images uploaded to ECR as they still might be in use by versioned functions. During `sls remove`, the created ECR repository will be removed. During deployment, Framework will attempt to `docker login` to ECR if needed. Depending on your local configuration, docker authorization token might be stored unencrypted. Please refer to documentation for more details: https://docs.docker.com/engine/reference/commandline/login/#credentials-store
 
+## Instruction set architecture
+
+By default, Lambda functions are run by 64-bit x86 architecture CPUs. However, using arm64 architecture (AWS Graviton2 processor) may result in better pricing and performance.
+
+To switch all lambdas to AWS Graviton2 processor, configure `arch` at `provider` level as follows:
+
+```yml
+provider:
+  ...
+  arch: arm64
```

Comment:
```suggestion
  architecture: arm64
```

The word isn't really long so we might as well be explicit for clarity? Additionally the [CloudFormation setting](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-architectures) is `Architectures`, not `Arch`/`Archs` so we gain a bit in consistency.

Reply:
I actually followed the spec from the issue, which was confirmed as accepted.

Anyway I also thought about that, and I agree that using `architecture` is probably better

Reply:
👍  ok I understand better, I thought the original post (top comment of the issue) was just a suggestion. So I'm guessing for anything discussed on GitHub the practice is to discuss and update the top comment?

Reply:
Yes, issue serves as specification then